applications/plugins/CollisionOBBCapsule/src/CollisionOBBCapsule/detection/intersection/OBBIntersection.h

```diff
@@ -84,13 +80,9 @@ class COLLISIONOBBCAPSULE_API RigidMeshDiscreteIntersection : public core::colli
     }
 
     template <class Elem1, class Elem2>
-    bool testIntersection(Elem1& e1, Elem2& e2) {
+    bool testIntersection(Elem1& e1, Elem2& e2, const core::collision::Intersection* intersection) {
         return BaseIntTool::testIntersection(e1, e2, intersection->getAlarmDistance());
     }
-
-protected:
-    NewProximityIntersection* intersection;
```

Comment:
Why don't you keep track of the interaction anymore ? Is it a mechanical consequence of the deprecation or a change that is introduced by the PR ?

Reply:
https://github.com/sofa-framework/sofa/pull/4583 🫢
The whole point is to remove this pointer actually 😅